Yank/yamlbuild.py

```diff
@@ -1177,14 +1178,17 @@ class YamlBuilder:
     def yank_options(self):
         return self._isolate_yank_options(self.options)
 
-    def __init__(self, yaml_source=None):
+    def __init__(self, yaml_source=None, platform_name=None):
```

Comment:
The only solution to this, is to *not* have command line arguments at all, and have everything in the YAML script only, which I'm still open too. I don't have a strong opinion about it.